aten/src/ATen/native/TensorIterator.cpp

```diff
@@ -842,35 +842,6 @@ void TensorIterator::compute_shape(const TensorIteratorConfig& config) {
   }
 }
 
-void TensorIterator::resize_outputs(const TensorIteratorConfig& config) {
-  if (config.static_shape_.has_value()) {
-    return;
-  }
-  // Outputs cannot be broadcasted. Check that the shape of the outputs matches
```

Comment:
I noticed you didn't preserve this comment; intentional? I think it's still true that outputs cannot be broadcasted...

Reply:
No, it's not intentional, I restore the comment.